### ISSUE TITLE
Sphinx Gallery expansion. Include more figures.

### ISSUE THREAD

**Issue by kelle** [astropy/astropy-tutorials#132]
[thread started on astropy-dev](https://groups.google.com/forum/#!topic/astropy-dev/RYwaRAa0fjE) by @adrn. Below list based on F2F meeting between @kelle and @adrn. 

- [ ] move example gallery to its own repo within the astropy organization.
- [ ] figure out how to version the example gallery to stay in step with astropy core. release in parallel? @bsipocz ?
- [ ] figure out new subdomain name. Want one subdomain which will eventually hold Examples, Tutorials, and Guides. 
    - stick with tutorials.astropy.org which people already know? or maybe change to learn.astropy.org? @eblur ?
- [ ] once above things are worked out, send update to astropy-dev.
- [ ] Update links to new example gallery location.
- [ ] make CONTRIBUTE doc and circulate so people can start adding things.
- [ ] make existing figures in tutorials examples so the two formats (Examples <-> Tutorials) x-link.

Stretch goals:
- guidelines/ best practices for visualization.
- integrate tutorials into this repo

**Comment by kelle** [comment]
I like the idea of `learn.astropy.org` as being a landing page with pointers to the Examples, Tutorials, and Guides. Not that many people know about `tutorials.astropy.org`

**Comment by bsipocz** [comment]
I have to dig into the details (and don't have time for it *right* now), but think that it should be definitely possible to trigger a travis build of a separate repo and probably even report back the result via a github integration. However if the repo lives under astropy org, we may not want to do this for every commit as it will use the same CI resources as if it were left in core.

**Comment by kelle** [comment]
this convo is out of date. close?

**Comment by adrn** [comment]
Irrelevant now, yep!
